ecosystem/sep-0006.md

```diff
@@ -536,7 +536,7 @@ Name | Type | Description
 `destination_asset` | string | The code of the on-chain asset the user wants to get from the Anchor after doing an off-chain deposit. The value passed must match one of the codes listed in the [/info](#info) response's `deposit-exchange` object.
 `source_asset` | string | The off-chain asset the Anchor will receive from the user. The value must match one of the `asset` values included in a [`SEP-38 GET /prices?buy_asset=stellar:<destination_asset>:<asset_issuer>`](sep-0038.md#get-prices) response using [SEP-38 Asset Identification Format](sep-0038.md#asset-identification-format).
 `quote_id` | string | (optional) The `id` returned from a `SEP-38 POST /quote` response. If this parameter is provided and the user delivers the deposit funds to the Anchor before the quote expiration, the Anchor should respect the conversion rate agreed in that quote. If the values of `destination_asset`, `source_asset` and `amount` conflict with the ones used to create the [SEP-38] quote, this request should be rejected with a `400`.
-`amount` | string | The amount of the source_asset the user would like to deposit to the anchor's off-chain account. This field may be necessary for the anchor to determine what KYC information is necessary to collect.
+`amount` | string | The amount of the 1source_asset1 the user would like to deposit to the anchor's off-chain account. This field may be necessary for the anchor to determine what KYC information is necessary to collect. If `quote_id` was used, `amount` should be equals the quote's `sell_amount`.
```

Comment:
In that case, we'll need to change the item 11 of the [Detailed Sending Anchor Flow](https://github.com/stellar/stellar-protocol/blob/master/ecosystem/sep-0031.md#detailed-sending-anchor-flow), so it changes from:
```text
11. The Sending Anchor submits the payment transaction to Stellar using the information provided in the `POST /transactions` response.
```
to 
```text
11. The Sending Anchor submits the payment transaction to Stellar using the information provided in the `GET /transactions/:id` response.
```

Reply:
Well, part of the direction I wanted to set here was to **not allow** the user's desired amount to be changed by the anchor (focused on STRICT_SEND).

So the next step seems to be answering the question **do we want to allow Anchors to change the amount_in based on the desired amount + fees?**

If the answer is YES, we need to update both SEP-31 and SEP-6 so that after creating the transaction, the client (or sending anchor) needs to query the transaction by ID in order to figure out the amount they should be depositing

WDYT? @JakeUrban, @leighmcculloch, @lijamie98, @stfung77.